app/models/manageiq/providers/vmware/infra_manager/inventory/collector.rb

```diff
@@ -102,6 +102,7 @@ def full_refresh(vim, property_filter)
     save_inventory(persister)
 
     self.last_full_refresh = Time.now.utc
+    clear_cis_taggings
```

Comment:
After the full refresh is done we can clear cached inventory which is only used by full-refresh (tags/categories aren't updated with WaitForUpdates so it is immediately stale)